src/app/Dependencies.tsx

```diff
@@ -0,0 +1,123 @@
+import { useState, useEffect, Fragment, FC } from 'react';
+import { jsx } from '@emotion/core';
+import ReactFlow, { ReactFlowProvider, isNode } from 'react-flow-renderer';
+import * as dagre from 'dagre';
+import { v4 as uuid } from 'uuid';
+import { getDependencyMap } from './commands';
+import { useStore } from './store';
+
+interface ExtensionItemProps {
+  dependencies: Record<string, string[]>;
+}
+
+const DisplayDependencies: FC<ExtensionItemProps> = ({ dependencies }) => {
+  const [elements, setElements] = useState([]);
+
+  useEffect(() => {
+    getData();
+  }, []);
+
+  const position = { x: 0, y: 0 };
+  const edgeType = 'smoothstep';
+  const dagreGraph = new dagre.graphlib.Graph();
+  dagreGraph.setDefaultEdgeLabel(() => ({}));
+
+  const nodeWidth = 172;
+  const nodeHeight = 36;
+
+  function getData() {
```

Comment:
Not sure why this is an extra function - just bring it directly inside the `useEffect` hook. The way it is we just have one unnecessary allocation per render.